include/lattice_field.h

```diff
@@ -7,6 +7,7 @@
 #include <comm_quda.h>
 #include <util_quda.h>
 #include <object.h>
+#include <cuda_runtime.h>
```

Comment:
Is this the right place to put that? But I don't really care, just curious.

Reply:
Yes, it is needed.  It's needed here because this include file uses CUDA runtime structs, such as events and streams, but this file is included in pure `cpp` code.  I found this issue with some header reorganization, that this include was missing but needed.